### PR TITLE
Package name Refactor

### PR DESCRIPTION
**Pull request by rairatne**
This includes changes to rename package to
"com.intel.sample.dms" as the object detection and dms both where having same package name

Signed-off-by: Ratnesh Kumar Rai <ratnesh.kumar.rai@intel.com>

### CODE REVIEW

File: android/app/src/main/java/com/intel/sample/dms/CameraActivity.java [hunk 1]
```diff
@@ -46,25 +45,23 @@
...
 import android.view.View;
 import android.view.WindowManager;
 import android.widget.Button;
-import android.widget.CompoundButton;
-import android.widget.ImageView;
-import android.widget.LinearLayout;
 import android.widget.ProgressBar;
 import android.widget.TextView;
 import android.widget.Toast;
-import com.google.android.material.bottomsheet.BottomSheetBehavior;
```
Comment (akodanka) [inline android/app/src/main/java/com/intel/sample/dms/CameraActivity.java:62]: These got removed as part of the cleanup refactor ?

**Review by akodanka — APPROVED** [review]
LGTM

File: android/app/src/main/java/com/intel/sample/dms/CameraActivity.java (hunk 1 above)
Comment (rairatne) [reply android/app/src/main/java/com/intel/sample/dms/CameraActivity.java:62]: Yes seems we are not using it, earlier there was a pull-up feature to see inference time and we are not using it for DMS app

Comment (akodanka) [reply android/app/src/main/java/com/intel/sample/dms/CameraActivity.java:62]: yup, I had removed it, but maybe the cleanup didn't remove all dependencies back then.

**Pull request merged**